packages/expo-updates/e2e/fixtures/project_files/eas.json

```diff
@@ -19,51 +25,40 @@
     "updates_testing_debug": {
       "extends": "base",
       "env": {
-        "EX_UPDATES_NATIVE_DEBUG": "1",
-        "NO_FLIPPER": "1"
+        "EX_UPDATES_NATIVE_DEBUG": "1"
       },
       "android": {
-        "image": "ubuntu-22.04-jdk-17-ndk-r21e",
+        // Not necessarily to upload the archive for updates_testing_debug, upload eas.json instead.
+        "applicationArchivePath": "eas.json",
         "gradleCommand": ":app:assembleDebug :app:assembleAndroidTest -DtestBuildType=debug",
-        "withoutCredentials": true,
-        "resourceClass": "large"
+        "withoutCredentials": true
       },
       "ios": {
+        // Not necessarily to upload the archive for updates_testing_debug, upload eas.json instead.
+        "applicationArchivePath": "eas.json",
```

Comment:
I thought that `buildArtifactPaths` below was what controlled the archive contents -- it would be good to get the logs for all the different test flows, if that is possible.

Reply:
i didn't touch the `buildArtifactPaths` so we can still get the test log. for example in https://expo.dev/accounts/expo-ci/projects/updates-e2e/builds/375aab07-c03e-4d4b-835c-fa97353bd7d6. the download build turns to eas.json but download artifacts still has detox log